### ISSUE TITLE
Multilevel side-menu

### ISSUE THREAD

**Issue by VladimirChumack** [gethinode/hinode#178]
**Is your feature request related to a problem? Please describe.**

I am working on a documentation site for multiple products (more than 2000 pages in total)

**Describe the solution you'd like**

I want to create a side menu for every product like

└── product
    ├── folder 1
    │   ├── subfolder 1
    │   ├── subfolder 2    
    │   └── subfolder 3  
    ├── folder 2       

Folders and subfolders may have multiple pages and subfolders 

I have tried modifying the menu yaml file without any luck 

- title: Product 1
     - title: Title1
       pages:
         - title: Introduction
         - title: Requirements
     - title: Title2
       pages:
         - title: Page 1
         - title: Page 2

Thank you

**Comment by markdumay** [comment]
I released `v0.11.1` with support for nested menus. You can define nesting in the following way:

```yml
- title: A
  pages:
    - title: B
      pages:
        - title: C
          pages:
            - title: First
            - title: Second
            - title: Third
```

There might be still a few bugs to iron out. Looking forward to your feedback!

**Comment by VladimirChumack** [comment]
Thank you. That will help us a lot

**Comment by markdumay** [comment]
I published a new release `v0.11.2`. The [sidebar documentation](https://gethinode.com/docs/0.9/configuration/navigation/#sidebar-navigation) has been updated too. The following changes have been made:
- Content pages no longer need a `group` parameter in their front matter
- Sections now support optional versioning

**Comment by markdumay** [comment]
I'll close the issue for now. Feel free to re-open it in case you encounter any issues.
